components/src/dynamo/vllm/tests/test_vllm_worker_factory.py

```
@@ -49,6 +50,182 @@ def _make_config(**overrides) -> Mock:
+    engine_client = Mock()
+    vllm_config = SimpleNamespace(
+        additional_config={},
+        cache_config=SimpleNamespace(num_gpu_blocks=1),
+        model_config=SimpleNamespace(max_model_len=1024),
+        shutdown_timeout=5.0,
+    )
+    engine_setup: EngineSetupResult = (
+        engine_client,
+        vllm_config,
+        Mock(),
+        "/tmp/prometheus",
```

Ruff (0.16.1)

[error] Probable insecure usage of temporary file or directory: "/tmp/prometheus"

(S108)